packages/custom_lint/lib/custom_lint.dart

```diff
@@ -154,7 +154,7 @@ void _renderLints(
   for (final error in errors) {
     stdout.writeln(
       '  ${_relativeFilePath(error.location.file, workingDirectory)}:${error.location.startLine}:${error.location.startColumn}'
-      ' • ${error.message} • ${error.code}',
+      ' • ${error.message} • ${error.code} • ${lint.severity.name}',
```

Comment:
The `lint` variable doesn't seem to exist. Maybe you're looking for the `error` variable?